cmd/yaman/README.md

```diff
@@ -0,0 +1,151 @@
+$ sudo yet Another (container) MANager
+
+## Commands
+
+**👋 Make sure to [follow these instructions](../../README.md#building-this-project) first.**
+
+⚠️ You must have `root` privileges to use `yaman` because the tool needs to mount the "rootfs" of a container as an Overlay FS. You can use `sudo yaman`.
```

Comment:
```suggestion
⚠️ You must have "root" privileges to use `yaman` because the tool needs to mount the "rootfs" of a container as an Overlay FS. You can use `sudo yaman` as shown in the next sections.
```